### DISCUSSION TITLE
Simple scraper with ctk :D

### DISCUSSION THREAD

**Discussion by guilhermecugler** [TomSchimansky/CustomTkinter#2141] [Showcase]
Hey, I made a scraper and decided to do a simple GUI and CTk worked really well!

Repository: [Scrape Casa dos Dadosr](https://github.com/guilhermecugler/Scraper-Casa-dos-Dados/)

<img src="https://github.com/guilhermecugler/Scraper-Casa-dos-Dados/raw/main/images/preview.png" alt="Preview">

**Comment by SomeRandomUsern4m3** [comment]
Cool
